DeepSpeech.ipynb

```diff
@@ -106,7 +107,14 @@
     "do_fulltrace = bool(len(os.environ.get('ds_do_fulltrace', '')))\n",
     "\n",
     "if do_fulltrace:\n",
-    "    check_cupti()"
+    "    check_cupti()\n",
+    "\n",
+    "track_gpu_usage = bool(len(os.environ.get('ds_track_gpu_usage', '')))\n",
+    "gpu_usage_charts = os.environ.get('ds_gpu_usage_charts', 'gpu_usage_%%s_%d.png' % int(time.time()))\n",
+    "\n",
+    "if track_gpu_usage:\n",
+    "    gu = GPUUsage()\n",
+    "    gu.start()"
```

Comment:
I guess it's not there, but here[[1](https://indico.io/blog/tensorflow-data-input-part2-extensions/)] where a custom `QueueRunner` threads are started after the built in `QueueRunner` threads
```python
# start the tensorflow QueueRunner's
tf.train.start_queue_runners(sess=sess)
# start our custom queue runner's threads
custom_runner.start_threads(sess)
```
So I'm guessing/extrapolating that we should also start any custom threads after the built in `QueueRunner` threads.

I don't have any proof that this is the correct time to start all custom threads, but this suggests that it's a good guess.